setup.py

```diff
@@ -23,10 +22,13 @@ def get_build():
 
     return str(build)
 
+
 setup(
     name="pylast",
     version="1.0." + get_build(),
     author="Amr Hassan <amr.hassan@gmail.com>",
+    install_requires=['pyyaml'],
```

Comment:
yep, sorry, I got a pyyaml error when I ran tox and wasn't playing close attention.